src/ethereum_spec_tools/evm_tools/t8n/env.py

```diff
@@ -114,12 +116,25 @@ def read_excess_blob_gas(self, data: Any, t8n: "T8N") -> None:
             self.parent_excess_blob_gas + self.parent_blob_gas_used
         )
 
-        target_blob_gas_per_block = t8n.fork.TARGET_BLOB_GAS_PER_BLOCK
+        if "currentTargetBlobsPerBlock" in data:
```

Comment:
For post-Prague forks, `currentTargetBlobsPerBlock` seems to be a mandatory field. So, if that is not available in the data, I believe the tools should throw an exception

Reply:
As currently written, we don't take into account what the current fork is at this point. This is an issue for the t8n refactor/rewrite to deal with.